### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Priyak5**
Changes to use Leaflet's built in L.map to initialize a map.

### CODE REVIEW

**Comment by sashadev-sky** [comment]
@Priyak5 great job! Thank you! I think there were 3 files that were overlooked from the original issue description. Can you update those too please?

**Comment by Priyak5** [comment]
Hey @sashadev-sky Thank you! 
I updated all the files and made PR's for them as well.

**Comment by sashadev-sky** [comment]
@Priyak5 and also we will make a PR template soon, but if you could link the original issue in the PR description body like this #182 that would be great.

**Comment by Priyak5** [comment]
@sashadev-sky okay I will.

**Comment by sashadev-sky** [comment]
@Priyak5 Ok can you push all of that work here? 1 branch per issue please

**Comment by Priyak5** [comment]
@sashadev-sky Sorry, I didn't know. On it. 
Thank you.

**Comment by sashadev-sky** [comment]
All good! Yeah thats what the FTOs are made for pretty much just to show the workflow. You will be helping people soon too :)

**Comment by Priyak5** [comment]
@sashadev-sky Thank you and updated all the files in this branch. :)

**Comment by sashadev-sky** [comment]
Perfect! Thank you! One more thing: can you please run `grunt`  in your CLI to concatenate your src files into the dist file, and then push the dist file that has changes staged up? We are working on removing this part from the process but for now it is necessary

**Comment by Priyak5** [comment]
@sashadev-sky  yeah sure. :)

**Comment by sashadev-sky** [comment]
Oops meant to edit my comment but accidentally deleted. Ok this PR is good to go!

**Comment by sashadev-sky** [comment]
@jywarren I think this is one good to merge!

File: examples/select.html [hunk 1]
```diff
@@ -35,7 +35,7 @@
     (function(){
 
       // basic Leaflet map setup
-      map = new L.map('map').setView([51.505, -0.09], 13);
+      map =L.map('map').setView([51.505, -0.09], 13);
```
Comment (rexagod) [inline examples/select.html]:
```suggestion
      map = L.map('map').setView([51.505, -0.09], 13);
```

**Comment by rexagod** [comment]
@sashadev-sky This looks good to me! @Priyak5 I recommend you look around this library, you'll definitely find it interesting, and we're here to help you otherwise. Thanks all! :+1:

**Comment by sashadev-sky** [comment]
@Priyak5 you have a merge conflict now, there was just a big merge so you need to do a rebase. Please follow these steps under "additional workflows" and let me know if you have any questions:
https://publiclab.org/wiki/contributing-to-public-lab-software#Rewinding+the+master+branch

Also please note the branch here is called `main` not `master`

Sorry for the bad timing of the merge! Probably should have opened this issue after merge

**Comment by Priyak5** [comment]
Hey , @sashadev-sky .
If the merge conflict is only in README.md can I not just update it according to the new merge?
If not , I can try what is given in the link.

**Comment by sashadev-sky** [comment]
Definitely need a rebase because all of the files you were working on have been changed. But that could get complicated for you because it is your first time working with the repo. I can try to pull your PR and rebase it for you tonight :) Typically we don't do it for people but it makes sense in this situation

**Comment by Priyak5** [comment]
@sashadev-sky Thank you! :)

**Comment by sashadev-sky** [comment]
@Priyak5 ok done! Thank you for your work!! 

@jywarren this is ready to merge

**Comment by jywarren** [comment]
Thank you so much! This is super 😄 👍 🎉

**Pull request merged**